pkg/ali/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "ali",
+  "description": "UI for Alibaba Cloud cluster provisioning",
```

Comment:
```suggestion
  "description": "Create Alibaba Cloud clusters in Rancher",
```

would follow the current pattern

<img width="1395" height="480" alt="Image" src="https://github.com/user-attachments/assets/e0b53f9e-e84a-467c-9b76-db3f6822b980" />